src/targets/python/requests/client.ts

```diff
@@ -181,8 +181,7 @@ export const requests: Client<RequestsOptions> = {
     push(request);
     blank();
 
-    // Print response
-    push('print(response.json())');
+    push('print(response.text)');
```

Comment:
This is going to change all `requests` code snippets we show in the API Explorer to do this instead of `response.json()` but that's probably fine.